cypress/e2e/billingv2.cy.ts

```diff
@@ -17,7 +17,7 @@ describe('Billing', () => {
         )
         cy.contains('.LemonModal .LemonButton', 'Unsubscribe').click()
 
-        cy.get('[data-attr=upgrade-card-product_analytics]').should('be.visible')
+        cy.get('.LemonModal').should('not.exist')
```

Comment:
seems like we'd want to make sure`/api/billing-v2/deactivate?products=product_analytics` actually gets hit here